pkg/crc/errors/multierror.go

```diff
@@ -26,3 +29,34 @@ func (m MultiError) ToError() error {
 	}
 	return fmt.Errorf(strings.Join(errStrings, "\n"))
 }
+
+// RetriableError is an error that can be tried again
+type RetriableError struct {
+	Err error
+}
+
+func (r RetriableError) Error() string {
+	return "Temporary Error: " + r.Err.Error()
+}
+
+// RetryAfter retries a number of attempts, after a delay
+func RetryAfter(attempts int, callback func() error, d time.Duration) (err error) {
+	m := MultiError{}
+	for i := 0; i < attempts; i++ {
+		if i > 0 {
+			log.Debugf("retry loop %d", i)
```

Comment:
Any reason we are using libmachine log package here, instead of crc logging package? The only problem i could see is when this is used outside of the machine package, where we are not setting libmachine logging!

Reply:
@anjannath Done.